### DISCUSSION TITLE
Move to official Stencil Playwright setup

### DISCUSSION THREAD

**Discussion by mJarsater** [scania-digital-design-system/tegel#600] [Ideas]
https://stenciljs.com/docs/testing/playwright/overview#set-up

The team should probably look into moving away from the unofficial stencil/playwright pkg that is currently used to instead use the official @stencil/playwright supported by Stencil.

**Comment by theJohnnyMe** [comment]
Nice to share this with us Martin. As I saw, it is still in experimental phase and breaking changes might be introduced anytime. Considering the size of the team and prios, I think transfer to officially supported version of Playwright from Stencil will happen once package gets first stable version.

**Comment by joao-goncalves-vwgds** [comment]
Hi @mJarsater, we have replaced the unofficial `stencil-playwright` package with the official `@stencil/playwright` one, on [this PR](https://github.com/scania-digital-design-system/tegel/pull/2024/changes#diff-0b810c38f3c138a3d5e44854edefd5eb966617ca84e62f06511f60acc40546c7R414).
Thanks for your input!
